# <span style="color:rgb(160,0,86)">Schätzverfahren</span>

***

## <span style="color:rgb(160,0,86)">Lernziele</span>

- Sie kennen Schätzverfahren für Parameter der Grundgesamtheit
- Sie können Punktschätzer von Intervallschätzen unterscheiden
- Sie können für die wichtigen Parameter deiser Schätzer berechnen

***

### <span style="color:rgb(160,0,86)">Was sind Schätzverfahren?</span>

Die **Schätztheorie** ist ein Teilgebiet der Statistik. Mit mathematischen Verfahren sollen *unbekannte Verteilungsparameter* (mittlere Lage, Streuung, etc.) einer Grundgesamtheit auf Basis von Zufallsstichproben *geschätzt werden*. Es wird zwischen **Punktschätzugen** und **Intervallschätzungen** unterschieden.

<span style="color:rgb(160,0,86)">***Punktschätzer:***</span> Ein Punktschätzer gibt **einen einzelnen Wert** als Schätzung für einen unbekannten Parameter an.

- *Beispiel:* Der **Mittelwert** $\pmb{\bar{x}}$ einer Stichprobe als Schätzung für den wahren **Mittelwert** $\pmb{\mu}$ der Grundgesamtheit. Die **relative Häufigkeit** $\pmb{h(\;)}$ in einer Stichprobe als Schätzung für die tatsächliche **Wahrscheinlichkeit** $\pmb{P(\;)}$ in der Grundgesamtheit.
- *Vorteil:* Einfach und direkt interpretierbar.
- *Nachteil:* Enthält keine Information über die Unsicherheit der Schätzung.

Ein Punktschätzer ist wie eine einzelne Zahl auf einer Zielscheibe – er kann genau treffen oder danebenliegen.


<span style="color:rgb(160,0,86)">***Intervallschätzer:***</span> Ein Intervallschätzer gibt **ein Intervall** an, in dem der wahre Parameter mit einer bestimmten Wahrscheinlichkeit liegt.

- *Beispiel:* Ein **95%-Konfidenzintervall** $\pmb{[x_{\text{min}},x_{\text{max}}]}$ für den Mittelwert gibt an, in welchem Bereich der wahre Mittelwert der Grundgesamtheit mit einer Wahrscheinlichkeit von 95% liegt.
- *Vorteil:* Enthält Informationen über die Unsicherheit der Schätzung.
- *Nachteil:* Komplexer als ein einzelner Punktwert.

Ein Intervallschätzer ist wie ein Zielfernrohr, das eine gewisse Streuung zulässt – es gibt nicht nur eine Zahl, sondern einen Bereich möglicher Werte.

### <span style="color:rgb(160,0,86)">Wie werden Punktschätzer festgelet?</span>

Häufig werden Punktschätzer mit einer der folgenden Methoden bestimmt.

<span style="color:rgb(160,0,86)">***Kleinste-Quadrate-Schätzung LSE:***</span>

Man macht keine Annahmen über die Verteilung in der Grundgesamtheit und zieht eine Zufallsstichprobe $x_1,x_2,\ldots,x_n$. 

- *Ziel:* Die Summe der quadrierten Fehler zwischen den beobachteten Werten und dem vorhergesagten Wert werden minimiert. Der **Schätzer** $\pmb{\hat{\mu}}$ für den Mittelwert $\mu$ einer Grundgesamtheit hat also die Eigenschaft, dass für die gegebene Stichprobe die Zahl $$(x_1-\pmb{\hat{\mu}})^2 + (x_2-\pmb{\hat{\mu}})^2 + (x_3-\pmb{\hat{\mu}})^2 + \cdots + (x_n-\pmb{\hat{\mu}})^2$$ minimal ist. Mit den Methoden der Analysis folgt, dass $$\pmb{\hat{\mu}} = \frac{1}{n}(x_1 + x_2 + x_3 + \cdots + x_n)$$ gerechnet werden muss. Der Mittelwert $\mu$ wird also mit dem **arithmetischen Mittel** geschätzt.


<span style="color:rgb(160,0,86)">***Maximum-Likelihood-Schätzung MLE:***</span>

Man nimmt an, dass die Daten der Grundgesamtheit aus einer bestimmten Verteilung stammen (z.B. Normalverteilung, Binomialverteilung, Poissonverteilung etc.) und zieht eine Zufallsstichprobe $x_1,x_2,\ldots,x_n$. Der Parameter $\mu$ dieser Verteilung ist aber unbekannt.  

- *Ziel:* Die Likelihood-Funktion $L(\mu|x_1,x_2,x_3,\ldots,x_n)$ gibt an, wie wahrscheinlich die Daten $x_1,x_2,\ldots,x_n$ der Stichprobe für verschiedene Parameterwerte $\mu$ der gewählten Verteilung sind. Den Schätzer $\pmb{\hat{\mu}}$ für den Parameter $\mu$ bestimmt man so, dass die Likelihood-Funktion $L$ am grössten ist. Das bedeutet also, dass der Parameter $\pmb{\hat{\mu}}$ die beobachteten Daten am wahrscheinlichsten erklärt. Wenn man zum Beispiel davon aus geht, dass die Daten Normalverteilt sind, dann folt mit den Methoden der Analysis, dass $$\pmb{\hat{\mu}} = \frac{1}{n}(x_1 + x_2 + x_3 + \cdots + x_n)$$ gerechnet werden muss. Der Mittelwert $\mu$ muss also auch mit dem **arithmetischen Mittel** geschätzt werden.


### <span style="color:rgb(160,0,86)">Welche Eingenschaften sollten Punktschätzer haben?</span>

- Eine Schätzer heisst **konsistent**, wenn mit zunehmendem Stichprobenumfang $n$ grosse Abweichungen zwischen dem wahren Parameter und dem geschätzten Parameter immer seltener werden.

- Ein Schätzer heisst **erwartungstreu**, wenn er im Mittel über alle Stichproben den wahren Parameter richtig schätzt.

Der *Stichprobenmedian* ist der Median der geordneten Stichprobe. Er ist im Allgemeinen **nicht erwartungstreu** für den wahren Median der Grundgesamtheit, dennoch ist er **konsistent**, das heisst, für grosse Stichproben nähert er sich dem wahren Median an.

### <span style="color:rgb(160,0,86)">Aufgabe 1</span>

Wir betrachten eine kleine Grundgesamtheit von 5 Personen:

$$\begin{array}{c|cccc}
&\text{PKW-Besitz }Y&\text{Einkünfte }X\\ \hline 
\text{Person A}&1&800\\
\text{Person B}&0&600\\
\text{Person C}&1&900\\
\text{Person D}&1&800\\
\text{Person E}&0&700\\
\end{array}$$

Aus dieser Grundgesamtheit wird eine Stichprobe $(y_1,x_1),(y_2,x_2)$ vom Umfang $n=2$, um den **Anteil PKW-Besitzer** $\pmb{p}$, die **durchschnittlichen Einkünfte** $\pmb{\mu}$ und die **Varianz der Einkünfte** $\pmb{\sigma^2}$ zu schätzen. 

Es werden folgende Schätzer verwendet:
- $\displaystyle\pmb{\hat{p}}=\frac{1}{2}(y_1+y_2)$
- $\displaystyle\pmb{\hat{\mu}}=\frac{1}{2}(x_1+x_2)\rule{0cm}{1.5cm}$
- $\displaystyle\pmb{\hat{\sigma}^2}=\frac{1}{2}\big((x_1-\pmb{\hat{\mu}})^2+(x_2-\pmb{\hat{\mu}})^2)\big)\rule{0cm}{1.5cm}$

1) Bestimmen Sie die wahren Parameter $\pmb{p}$, $\pmb{\mu}$ und $\pmb{\sigma^2}$ in der Grundgesamtheit.
2) Bestimmen Sie die Schätzungen der Parameter für alle möglichen Stichproben mit zurücklegen.
3) Bestimmen Sie die Schätzungen der Parameter für alle möglichen Stichproben ohne zurücklegen.
4) Sind die Schätzer bei 2) und 3) **erwartungstreu**?

In [49]:
import pandas as pd
import numpy as np
from itertools import combinations
from itertools import combinations_with_replacement

df = pd.DataFrame(data=np.array([[1,800],[0, 600],[1, 900],[1,800],[0,700]]), columns=np.array(["PKW-Besitz Y","Einkünfte X"]))

print("-------------------- Wahre werte ---------------------------")
print(f"Anteil PWK Besitzer {df["PKW-Besitz Y"].mean()}")
print(f"Durchschnittliche Einkünfte {df["Einkünfte X"].mean()}")
print(f"Varianz der Einkünfte {sum(np.abs(df["Einkünfte X"] - df["Einkünfte X"].mean()) / len(df["Einkünfte X"]))}")

-------------------- Wahre werte ---------------------------
Anteil PWK Besitzer 0.6
Durchschnittliche Einkünfte 760.0
Varianz der Einkünfte 88.0


In [50]:
meanIncomes = []
meanPKWs = []
meanMQDs = []
for (i, j) in combinations(df.index, 2):
    sample = df.loc[[i, j]]  # DataFrame mit 2 Zeilen
    meanPKW = sample["PKW-Besitz Y"].mean()
    meanIncome = sample["Einkünfte X"].mean()
    meanMQD = sum(np.abs(sample["Einkünfte X"] - meanIncome) / len(sample["Einkünfte X"]))
    meanIncomes.append(meanIncome)
    meanPKWs.append(meanPKW)
    meanMQDs.append(meanMQD)
    print(f"Anteil PWK Besitzer {meanPKW}")
    print(f"Durchschnittliche Einkünfte {meanIncome}")
    print(f"Varianz der Einkünfte {meanMQD}")
    print("\n")

meanPKWs = np.array(meanPKWs)
meanIncomes = np.array(meanIncomes)
meanMQDs = np.array(meanMQDs)
print(f"Erwartungstreu PWK-Besitz: {meanPKWs.mean() == df["PKW-Besitz Y"].mean()}")
print(f"Erwartungstreu Einkommen: {meanIncomes.mean() == df["Einkünfte X"].mean()}")
print(f"Erwartungstreu Varianz: {meanMQDs.mean() == sum(np.abs(df["Einkünfte X"] - df["Einkünfte X"].mean()) / len(df["Einkünfte X"]))}")

Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 700.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 850.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 800.0
Varianz der Einkünfte 0.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 150.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 700.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 0.0
Durchschnittliche Einkünfte 650.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 850.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 800.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 50.0


Erwartungstreu PWK-Besitz: True
Erwartungstreu Einkommen: True
Erwartungstreu Varianz: False


In [51]:
meanIncomes = []
meanPKWs = []
meanMQDs = []
for (i, j) in combinations(df.index, 2):
    sample = df.loc[[i, j]]  # DataFrame mit 2 Zeilen
    meanPKW = sample["PKW-Besitz Y"].mean()
    meanIncome = sample["Einkünfte X"].mean()
    meanMQD = sum(np.abs(sample["Einkünfte X"] - meanIncome) / len(sample["Einkünfte X"]))
    meanIncomes.append(meanIncome)
    meanPKWs.append(meanPKW)
    meanMQDs.append(meanMQD)
    print(f"Anteil PWK Besitzer {meanPKW}")
    print(f"Durchschnittliche Einkünfte {meanIncome}")
    print(f"Varianz der Einkünfte {meanMQD}")
    print("\n")

meanPKWs = np.array(meanPKWs)
meanIncomes = np.array(meanIncomes)
meanMQDs = np.array(meanMQDs)
print(f"Erwartungstreu PWK-Besitz: {meanPKWs.mean() == df["PKW-Besitz Y"].mean()}")
print(f"Erwartungstreu Einkommen: {meanIncomes.mean() == df["Einkünfte X"].mean()}")
print(f"Erwartungstreu Varianz: {meanMQDs.mean() == sum(np.abs(df["Einkünfte X"] - df["Einkünfte X"].mean()) / len(df["Einkünfte X"]))}")

Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 700.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 850.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 800.0
Varianz der Einkünfte 0.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 150.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 700.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 0.0
Durchschnittliche Einkünfte 650.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 1.0
Durchschnittliche Einkünfte 850.0
Varianz der Einkünfte 50.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 800.0
Varianz der Einkünfte 100.0


Anteil PWK Besitzer 0.5
Durchschnittliche Einkünfte 750.0
Varianz der Einkünfte 50.0


Erwartungstreu PWK-Besitz: True
Erwartungstreu Einkommen: True
Erwartungstreu Varianz: False


### <span style="color:rgb(160,0,86)">Welche Punktschätzer werden gebraucht?</span>

Gegeben sei eine Zufallsstichprobe $x_1,x_2,x_3,\ldots,x_n$ vom Umfang $n$ aus einer Grundgesamtheit. Die folgenden Schätzer sind *erwartungstreu*:
- Für den **Mittelwert** $\pmb{\mu}$ brauchen wir $$\pmb{\hat{\mu}}=\frac{1}{n}(x_1+x_2+x_3+\cdots + x_n) $$
- Für die **Varianz** $\pmb{\sigma^2}$ brauchen wir $$\pmb{\hat{\sigma}^2}=\frac{1}{\textcolor{red}{n-1}}\big((x_1-\bar{x})^2+(x_2-\bar{x})^2+(x_3-\bar{x})^2+\cdots + (x_n-\bar{x})^2\big) $$

Beachten Sie, dass die **mittlere quadratische Abweichung** $MQD$ *kein* erwartungstreuer Schätzer für die Varianz ist!

### <span style="color:rgb(160,0,86)">Wie werden Intervallschätzer berechnet?</span>

Während bei der Punktschätzung aus einer Zufallsstichprobe $x_1,x_2,x_3,\ldots,x_n$ nur ein punktueller Schätzwert ermittelt wird, informiert eine Intervallschätzung zusätzlich über den **Stichprobenfehler**. Sie stellt also einen Zusammenhang zwischen der Punktschätzung und dem Parameterwert der Grundgesamtheit her.

Bei der Intervallschätzung wird für einen unbekannten Paramter $\pmb{\theta}$ zunächst aus einer Stichprobe eine Punktschätzung $\pmb{\hat{\theta}}$ berechnet, um die herum dann ein (meist symmetrisches) **Konfidenzintervall** konstruiert wird: $$K=[\pmb{\hat{\theta}-\delta_n},\pmb{\hat{\theta}+\delta_n}]$$
Dabei wird der Stichprobenfehler $\pmb{\delta}$ so bestimmt, dass das Konfidenzintervall den unbekannten Parameter $\pmb{\theta}$ mit einer vorgegebenen Wahrscheinlichkeit überdeckt: $$P(\theta\in K) = \alpha$$ Diese Wahrscheinlichkeit $\alpha$  heisst **Konfidenzniveau**. Üblicherweise wereden in der Praxis Konfidenzniveaus von 95% oder 99% gewählt.

<center>
<img src="Bilder/Konfidenz.png" width="50%" />
</center>

In der klassischen Statistik wird der Stichprobenfehler $\pmb{\delta}$ für Mittelwerte mathematisch mit Formelen, insbesondere mit der $t$-Verteilung ermittelt: $$ \delta = \big\{(1-\alpha/2)\text{ Quantil von der Verteilung }t(n-1) \big\}\cdot \frac{\hat{\sigma}}{\sqrt{n}}\qquad $$
Dabei wird angenommen, dass die Daten *normalverteilt* sind, was für Stichprobenmittelwerte zutrifft.    


Eine moderne Möglichkeit (mit einem Computer) Konfidenzintervalle zu berechnen bietet das **Bootstrap-Verfahren**. Ein $\pmb{\alpha\%}$**-Konfidenzintervall** eines Parametres sollte im Durchschnitt in $\pmb{\alpha\%}$ der Fälle die gleichen Stichprobenschätzer abdecken, sofern zur Stichprobenerhebung dasselbe Verfahren verwednet wird.

Angenommen wir hätten eine Stichprobe der Grösse $n$ und interessierten uns für einen bestimmen Parameter. Dann können wir mit dem folgenden Algorithmus ein **Bootstrop-Konfidenzintervall** berechnen:

1) Ziehe eine Zufallsstichprobe (ein **Resample**) mit Zurücklegen der Grösse $n$ aus den Daten.
2) Erfasse den im Fokus stehende Parameter für das Resample.
3) Wiederhole die Schritte 1 und 2 viele Male.
4) Zur Ermittlung des $\pmb{\alpha\%}$**-Konfidenzintervall** müssen wir nun $$\big((100-\pmb{\alpha})/2\big)\%$$ der **geordneten** Werte der Resamples an beiden Enden abschneiden.
5) Die zwei Werte, an denen die **geordneten** Werte der Resamples gestutz wurden, bilden die entsprechenden Konfidenzschranken des $\pmb{\alpha\%}$**-Konfidenzintervall**.

### <span style="color:rgb(160,0,86)">Aufgabe 2</span>

Laden Sie die Datei *loans_income.csv* im Ordner Daten und nehmen je 1000  **Zufallsstichproben** vom Umfang $n=4,8,16,32,64,128$ aus diesen Daten. 

- Schätzen Sie den wahren Mittelwert und die wahre Standardabweichung der Löhne mit diesen Stichproben und vergleichen Sie diese Schätzungen mit den wahren Parametern.
- Bestimmen Sie mit Resamples basierend auf einer Stichprobe mit $n=128$ ein 95\%- und ein 99\%-Konfidenzintervall für den Mittelwert der Löhne.  

In [61]:
import pandas as pd
import numpy as np

df = pd.read_csv("Daten/loans_income.csv")

sampleSizes = [4,8,16,32,64,128]
for sampleSize in sampleSizes:
    meanIncome = []
    std = []
    for i in range(1000):
        sample = df['x'].sample(sampleSize, replace=False)
        meanIncome.append(sample.mean())
        std.append(sample.std(ddof=0))
    meanIncome = np.array(meanIncome)
    std =  np.array(std)
    print(f"Durchschnitt bei {sampleSize}-Umfag: {meanIncome.mean()}")
    print(f"Standardabweichhung bei {sampleSize}-Umfag: {std.mean()}")

print(f"Durchschnitt: {df['x'].mean()}")
print(f"Standardabweichung: {df['x'].std(ddof=0)}")

sample = df['x'].sample(128)
bootstrap_means = [sample.sample(128, replace=False) for _ in range(10000)]
lower_bound_1 = np.percentile(bootstrap_means, 2.5)
upper_bound_1 = np.percentile(bootstrap_means, 97.5)
lower_bound_2 = np.percentile(bootstrap_means, 0.5)
upper_bound_2 = np.percentile(bootstrap_means, 99.5)

print(f"95%-Konfidenzintervall: ({lower_bound_1:.4f}, {upper_bound_1:.4f})")
print(f"99%-Konfidenzintervall: ({lower_bound_2:.4f}, {upper_bound_2:.4f})")

Durchschnitt bei 4-Umfag: 69194.34025
Standardabweichhung bei 4-Umfag: 25521.081182961887
Durchschnitt bei 8-Umfag: 68193.845
Standardabweichhung bei 8-Umfag: 28881.95695215174
Durchschnitt bei 6-Umfag: 68823.637
Standardabweichhung bei 6-Umfag: 28042.0443948122
Durchschnitt bei 32-Umfag: 68910.16190625
Standardabweichhung bei 32-Umfag: 31962.42711008456
Durchschnitt bei 64-Umfag: 68957.469828125
Standardabweichhung bei 64-Umfag: 32508.214234245224
Durchschnitt bei 128-Umfag: 68780.73246875
Standardabweichhung bei 128-Umfag: 32691.715218867263
Durchschnitt: 68760.51844
Standardabweichung: 32871.70664785295
95%-Konfidenzintervall: (30000.0000, 160000.0000)
99%-Konfidenzintervall: (22000.0000, 180000.0000)


### <span style="color:rgb(160,0,86)">Aufgabe 3</span>

Laden Sie die Datei *Palette.sav* im Ordner Daten. In dieser Datei sind **Gewicht**, **Höhe** und **Breite** von 750 Hühnereiern, die von Hühnern der Rasse Loheimer Braun gelegt wurden. Nehmen Sie je 100  **Zufallsstichproben** vom Umfang $n=4,8,16,32,64,128$ aus diesen Daten. 

- Schätzen Sie die wahren Mittelwerte und die wahren Standardabweichungen des Gewichts, der Höhe und der Bereite mit diesen Stichproben und vergleichen Sie diese Schätzungen mit den wahren Parametern.
- Bestimmen Sie mit Resamples basierend auf einer Stichprobe mit $n=128$ ein 95\%- und ein 99\%-Konfidenzintervall für die Mittelwerte und die Standardabweichungen. 

In [64]:
import pandas as pd
import numpy as np

df = pd.read_spss("Daten/Palette.sav")

attributes = ['Breite', 'Höhe', 'Gewicht']

for attribute in attributes:
    sampleSizes = [4,8,16,32,64,128]
    for sampleSize in sampleSizes:
        meanIncome = []
        std = []
        for i in range(100):
            sample = df[attribute].sample(sampleSize, replace=False)
            meanIncome.append(sample.mean())
            std.append(sample.std(ddof=0))
        meanIncome = np.array(meanIncome)
        std =  np.array(std)
        print(f"Durchschnitt bei {sampleSize}-Umfag: {meanIncome.mean()}")
        print(f"Standardabweichhung bei {sampleSize}-Umfag: {std.mean()}")
    
    print(f"Durchschnitt: {df[attribute].mean()}")
    print(f"Standardabweichung: {df[attribute].std(ddof=0)}")
    
    sample = df[attribute].sample(128)
    bootstrap_means = [sample.sample(128, replace=False) for _ in range(10000)]
    lower_bound_1 = np.percentile(bootstrap_means, 2.5)
    upper_bound_1 = np.percentile(bootstrap_means, 97.5)
    lower_bound_2 = np.percentile(bootstrap_means, 0.5)
    upper_bound_2 = np.percentile(bootstrap_means, 99.5)
    
    print(f"95%-Konfidenzintervall: ({lower_bound_1:.4f}, {upper_bound_1:.4f})")
    print(f"99%-Konfidenzintervall: ({lower_bound_2:.4f}, {upper_bound_2:.4f})")
    print("\n")


Durchschnitt bei 4-Umfag: 44.21999999999999
Standardabweichhung bei 4-Umfag: 0.9004884236899332
Durchschnitt bei 8-Umfag: 44.230375
Standardabweichhung bei 8-Umfag: 1.0597990534823492
Durchschnitt bei 16-Umfag: 44.224625
Standardabweichhung bei 16-Umfag: 1.1781235458944805
Durchschnitt bei 32-Umfag: 44.198375000000006
Standardabweichhung bei 32-Umfag: 1.1492209794142034
Durchschnitt bei 64-Umfag: 44.209578125
Standardabweichhung bei 64-Umfag: 1.1806672358470143
Durchschnitt bei 128-Umfag: 44.24335937500001
Standardabweichhung bei 128-Umfag: 1.1831201589339504
Durchschnitt: 44.2236
Standardabweichung: 1.1846756405587706
95%-Konfidenzintervall: (42.1000, 46.5000)
99%-Konfidenzintervall: (41.9000, 46.8000)


Durchschnitt bei 4-Umfag: 56.86525
Standardabweichhung bei 4-Umfag: 1.7040562125348016
Durchschnitt bei 8-Umfag: 56.8955
Standardabweichhung bei 8-Umfag: 1.868472583407963
Durchschnitt bei 16-Umfag: 57.02318749999999
Standardabweichhung bei 16-Umfag: 1.9692606385097011
Durchschnitt be

![HSLU](Bilder/LogoHSLU.png)